docs/references/files/fossa-yml.md

```diff
@@ -127,6 +130,11 @@ The `name:` and `release:` of the release group's release to add your project to
 
 If you choose to associate a project with a release group, you **must** supply both name and release.
 
+#### `project.labels:`
+The `labels` field allows you to add labels to projects so that you can classify certain projects how you would like. This adds a more flexible way to query for projects as opposed to assigning a project to a team.
```

Comment:
[nit] Maybe mention "querying in the FOSSA web UI." Someone who is mainly used to using the CLI may not understand where they would be querying.